<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SensorAnalysisDialog</class>
 <widget class="QDialog" name="SensorAnalysisDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>477</width>
    <height>283</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Sensor Analysis</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="9" column="0">
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="9" column="1">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="1">
    <widget class="QPushButton" name="mRunButton">
     <property name="text">
      <string>Run</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="1">
    <widget class="QLabel" name="mStatusMessage">
     <property name="text">
      <string>status</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QPushButton" name="mKillButton">
     <property name="text">
      <string>Kill</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="mTargetPlatformTypeButton">
     <property name="text">
      <string>Target Platform Type...</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QCheckBox" name="mUseDefaultRadarFrequencyCheckbox">
     <property name="text">
      <string>Default Radar Frequency</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QGroupBox" name="mOutputDirectoryGroupBox">
     <property name="title">
      <string>Output Directory</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="1">
       <widget class="QPushButton" name="mBrowseButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>124</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Browse...</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLineEdit" name="mOutputDirectoryEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" rowspan="8">
    <widget class="QGroupBox" name="mPlotOutputGroupBox">
     <property name="title">
      <string>Sensor Plot Selections</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="3" column="4">
       <widget class="QPushButton" name="mVerticalMapOptionsButton">
        <property name="text">
         <string>Options...</string>
        </property>
       </widget>
      </item>
      <item row="5" column="4">
       <widget class="QPushButton" name="mVerticalCoverageOptionsButton">
        <property name="text">
         <string>Options...</string>
        </property>
       </widget>
      </item>
      <item row="4" column="4">
       <widget class="QPushButton" name="mHorizontalCoverageOptionsButton">
        <property name="text">
         <string>Options...</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2" alignment="Qt::AlignHCenter">
       <widget class="QCheckBox" name="mGenerateVM">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Horizontal Map</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Show Plot</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Generate</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3" alignment="Qt::AlignHCenter">
       <widget class="QCheckBox" name="mPlotVM">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="3" alignment="Qt::AlignHCenter">
       <widget class="QCheckBox" name="mPlotHM">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="6" column="3" alignment="Qt::AlignHCenter">
       <widget class="QCheckBox" name="mPlotSM">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="2" alignment="Qt::AlignHCenter">
       <widget class="QCheckBox" name="mGenerateHCD">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="6" column="4">
       <widget class="QPushButton" name="mSphericalMapOptionsButton">
        <property name="text">
         <string>Options...</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Vertical Coverage</string>
        </property>
       </widget>
      </item>
      <item row="5" column="3" alignment="Qt::AlignHCenter">
       <widget class="QCheckBox" name="mPlotVCD">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="6" column="2" alignment="Qt::AlignHCenter">
       <widget class="QCheckBox" name="mGenerateSM">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="2" alignment="Qt::AlignHCenter">
       <widget class="QCheckBox" name="mGenerateHM">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Spherical Map</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Vertical Map</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3" alignment="Qt::AlignHCenter">
       <widget class="QCheckBox" name="mPlotHCD">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Horizontal Coverage</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2" alignment="Qt::AlignHCenter">
       <widget class="QCheckBox" name="mGenerateVCD">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QPushButton" name="mHorizontalMapOptionsButton">
        <property name="text">
         <string>Options...</string>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QCheckBox" name="mShowCalibrationDataCheckBox">
     <property name="text">
      <string>Show Calibration Data</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>mGenerateHM</tabstop>
  <tabstop>mPlotHM</tabstop>
  <tabstop>mHorizontalMapOptionsButton</tabstop>
  <tabstop>mGenerateVM</tabstop>
  <tabstop>mPlotVM</tabstop>
  <tabstop>mVerticalMapOptionsButton</tabstop>
  <tabstop>mGenerateHCD</tabstop>
  <tabstop>mPlotHCD</tabstop>
  <tabstop>mHorizontalCoverageOptionsButton</tabstop>
  <tabstop>mGenerateVCD</tabstop>
  <tabstop>mPlotVCD</tabstop>
  <tabstop>mVerticalCoverageOptionsButton</tabstop>
  <tabstop>mGenerateSM</tabstop>
  <tabstop>mPlotSM</tabstop>
  <tabstop>mSphericalMapOptionsButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
